web/scm/src/app/app-component/app.component.spec.ts

```diff
@@ -26,10 +26,10 @@ describe('AppComponent', () => {
     expect(app.title).toEqual('scm');
   });
 
-  it('should render title', () => {
+  it('should render para', () => {
```

Comment:
Can we have test name like: 'Should display hello message' ? I could't found a doc for this right now, but it's something like this.

Reply:
Okay, will change that